src/Module.php

```diff
@@ -80,6 +94,26 @@ public function setOtp(string $otp): void
         $this->otp = $otp;
     }
 
+    public function setSad(string $sad): void
+    {
+        $this->sad = $sad;
+    }
+
+    /**
+     * Sets the desired signing method to use on createSignature method
+     */
+    public function setSigningMethod($method)
+    {
+        $possible = ['otp','sad'];
+
+        if ( ! in_array($method, $possible ) )
```

Comment:
Please remove the spaces and take the opening bracket at the end of this line.

`if (!in_array($method, $possible)) {`